tests/stress/km/stress_tests_km.cpp

```diff
@@ -1428,15 +1429,12 @@ TEST_CASE("native_load_attach_detach_unload_random_v4_test", "[native_mt_stress_
     //   all threads executing in parallel with the bare minimum synchronization between them.
     //
     // - Addition of a dedicated thread continuously unloading/reloading the provider extension.
```

Comment:
```suggestion
    // - Addition of a dedicated thread continuously unloading/reloading the provider extension
```
nit: remove extra period